src/logic/dealer/dealer.js

```diff
@@ -39,11 +47,50 @@ function drawStartingCard() {
 
 // == Move Card ==
 
+/*
 export function handleMoveCard(card, sourcePile, destinationPile) {
     moveCard(card, sourcePile, destinationPile);
     setLeadingCard(card, destinationPile);
     // rollOverCard();
 }
+*/
+
+export function handleMoveCard(card, sourcePile) {
+    let destinationPile;
+    debugger;
```

Comment:
remove debugger, last time it costed us 5 points

Reply:
done